src/test/java/com/dataStructures/HashTableTest.java

```diff
@@ -0,0 +1,75 @@
+package src.test.java.com.dataStructures;
+
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+/**
+ *
+ * @author Rick
+ */
+public class HashTableTest {
+    
+    /**
+     * Test of add method, of class HashTable.
+     */
+    @org.junit.Test
```

Comment:
Since you've imported this class, you don't need to fully qualify it. You can simply write `@Test`

Reply:
Oc. thanks for the aclaration